packages/editor/src/components/post-publish-panel/index.js

```diff
@@ -27,6 +27,20 @@ export class PostPublishPanel extends Component {
 	constructor() {
 		super( ...arguments );
 		this.onSubmit = this.onSubmit.bind( this );
+		this.cancelButtonNode = createRef();
+	}
+
+	componentDidMount() {
```

Comment:
Unrelated: Haha, looks like these are very old components, some of the few that are still using the component class. Unrelated, but we should refactor this.

Reply:
Wow, I haven't seem a `componentDidMount` in a while 😂

Reply:
45 occurrences of `componentDidMount()` in the current codebase 😉